src/h3/api/_api_template.py

```diff
@@ -855,7 +855,7 @@ def experimental_h3_to_local_ij(self, origin, h):
         origin = self._in_scalar(origin)
         h = self._in_scalar(h)
 
-        i, j = _cy.experimental_h3_to_local_ij(origin, h)
+        i, j = _cy.cell_to_local_ij(origin, h)
```

Comment:
Same here. I'd like to make no changes to this file at this stage.

We will change all these function names (and the broader API as well), just not at this stage of the migration.